Testing/functional/e2e/cypress/integration/timeline/modals/covidModal.js

```diff
@@ -61,13 +61,26 @@ describe('Validate Modals Popup', () => {
         cy.get('[data-testid=covidModalText]').contains('Check the status of your COVID-19 test and view your result when it is available')
         cy.get('[data-testid=covidViewResultBtn]').should('be.visible').contains('View Result').click()
         cy.get('[data-testid=covidModal]').should('not.exist')
+        
```

Comment:
updated functional test for: "Dismiss by clicking "View Results" button will store the action date time in the database and also select the Covid-19 Tests filter. "